make/config/istio/istio-config-1.16.7.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+spec:
+  profile: "minimal"
+  meshConfig:
+    # Change the following line to configure the trust domain of the Istio cluster.
+    trustDomain: foo.bar
+  values:
+    global:
+      # Change certificate provider to cert-manager istio agent for istio agent
+      caAddress: cert-manager-istio-csr.istio-system.svc:443
```

```suggestion
      caAddress: cert-manager-istio-csr.cert-manager.svc:443
```
The e2e setup installs istio-csr in the cert-manager namespace.

---

I fixed this in https://github.com/cert-manager/istio-csr/pull/271/commits/5ba84bd77bdec205e0e9da0f6747f5bf867ae8aa.